api/src/main/resources/config/application-dev.yaml

```diff
@@ -6,14 +6,6 @@ spring:
     username: ${DB_USER:rosetta_db_admin}
     password: ${DB_SECRET:weakpwd#123_d}
     url: jdbc:postgresql://${DB_HOST:localhost}:${DB_PORT:5433}/${DB_NAME:test}?currentSchema=${DB_SCHEMA:preprod}
-    hikari:
```

Comment:
You could leave these settings as is, as it's only for dev environments. I hope it's more convenient to change these settings during the development phase from here.

Reply:
Done